code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -341,6 +341,50 @@
 	new /obj/item/stack/sheet/metal/fifty(src)
 	new /obj/item/stack/cable_coil(src)
 
+/obj/item/storage/box/large/glowstick/emergency
+	name = "emergency glowstick box"
+	desc = "A large box filled to the brim with cheap emergency glowsticks."
+
+/obj/item/storage/box/large/glowstick/emergency/populate_contents()
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
+	new /obj/item/flashlight/flare/glowstick/emergency(src)
```

Comment:
These boxes can hold a maximum of 7 items IIRC. You should limit the amount in each box to that.

Reply:
These are the large boxes, which hold a lot more items, I believe. There's less in here than the RND crate, which similarly uses large boxes.

Reply:
Eh, didnt see that but I'm not a big fan of them

Reply:
Large boxes hold 21 items and are very underutilized.